configs/retinanet/_base_/retinanet_reader.yml

```diff
@@ -0,0 +1,39 @@
+worker_num: 2
+TrainReader:
+  sample_transforms:
+  - Decode: {}
+  - RandomFlip: {prob: 0.5}
+  - Resize: {target_size: [800, 1333], keep_ratio: true, interp: 1}
```

Comment:
Better to use multi-scale training if you have enough GPUs

Reply:
OK, will add a multi-scale trained model.